cmd/agent/common/import.go

```diff
@@ -215,6 +218,20 @@ func copyFile(src, dst string, overwrite bool) error {
 		return err
 	}
 
+	ddID, err := user.LookupGroup("dd-agent")
+	fi, err := out.Stat()
```

Comment:
I think this won't change the user, so if the file was created by root it'll still be created by root here. Can you lookup the proper UID instead of grabbing it from the file that you're manipulating?

Reply:
That was actually my intent. I don't think we can assume that the user will want the file to be owned by the Agent user, but we just want to ensure the dd-agent user can read from it.

Reply:
I see, thanks for the input 🙇 I made the owner user and group owner of the file `dd-agent`